README.md

```diff
@@ -120,13 +222,23 @@ $ tree ./my-plugins/
     ├── postgresql-9.4-1206-jdbc41.jar
     └── sqlite-jdbc-3.8.11.2.jar
 ```
-* Start new image build using the prepared directory
+3. Start new image build using the prepared directory
 ```
 oc start-build kafka-connect --from-dir ./my-plugins/
 ```
-* You can find out the address of your local Docker repository (for example using `oc get imagestreams kafka-connect`)
-* Use your new image in the Kafka Connect deployment
+4. You can find out the address of your local Docker repository (for example using `oc get imagestreams kafka-connect`)
```

Comment:
The way I wrote this point is kind of strange. I think we should remove the brackets